services/QuillLMS/app/services/vitally_integration/serialize_vitally_sales_account.rb

```diff
@@ -49,25 +54,16 @@ def data
     }
   end
 
-  private def activities_per_student(active_students, activities_finished)
-    if active_students > 0
-      (activities_finished.to_f / active_students).round(2)
+  private def get_from_cache(key)
+    last_school_year = Date.today.year - 1
+    cached_data = $redis.get("school_id:#{@school.id}_vitally_stats_for_year_#{last_school_year}")
```

Comment:
I think you can utilize your new cache object: `cached_data = CacheVitallySchoolData.get(@school.id, last_school_year)`

Reply:
You'll notice that several of the comments in the PR are related to calling redis directly.  The value in abstracting that out into an object is that if we decide to change the name of the key at some point, we only have to change it in the new object rather than all of the individual calls.